src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj

```diff
@@ -8,9 +8,14 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Version>1.0.0</Version>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
@refortie remove line 11-16 you don't need it. (nit)
Please remove this next time.